### DISCUSSION TITLE
Welcome to forumoji Discussions!

### DISCUSSION THREAD

**Discussion by vercte** [vercte/forumoji#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RedToaster55** [comment]
Hello! I was thinking of getting this integrated into Scratch Addons. How easy would that be? Or perhaps there is already an extension I don't know about...

**Comment by vercte** [comment]
@mybearworld is working on a PR for this:
https://github.com/ScratchAddons/ScratchAddons/pull/6006
